VergeiOS/Views/WelcomeGuide/SelectPinViewController.swift

```diff
@@ -35,9 +38,14 @@ class SelectPinViewController: UIViewController, KeyboardDelegate {
         } else {
             self.pinTextField.addCharacter()
             
-            if (pin.count <= 6) {
+            if (pin.count <= 5) {
```

Comment:
Better use `pin.count < 6`, easier to read & understand

Reply:
Right?? wtf was I thinking 😝